### DISCUSSION TITLE
How do I compile a pydebug extension via pip?

### DISCUSSION THREAD

**Discussion by jpe** [pypa/setuptools#3996] [Q&A]
How do I compile an extension from pip for use with a python built with pydebug enabled? I used to use python setup.py build --debug

**Comment by abravalheri** [comment]
> How do I compile an extension from pip for use with a python built with pydebug enabled?

Am I correct in assuming that you are doing `pip install .`?

> I used to use python setup.py build --debug

I believe you can add the following section to your `setup.cfg` (just create one if you don't have it):

```ini
[build]
debug = True
```

You can also force `pip` to not hide any information by doing `pip -vvv install .`.
